main.cpp

```diff
@@ -66,9 +63,44 @@ void variableDeclarations()
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
     
-    
+    bool boolA = true;
+    bool boolB = false;
+    bool boolC = true;
+
+    char charA = 'A';
+    char charB = 'B';
+    char charC = 'C';
+
+    int numberA = 3;
+    int numberB = 37;
+    int numberC = 4137;
+
+    float floatA = 0.2f;
+    float floatB = 0.6736f;
+    float floatC = 4362.f; // can you put the f right after the decimal?
+
+    double doubleA = 0.2;
+    double doubleB = 39972403.3592;
+    double doubleC = 3.14159;
+
     
     ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(boolA, boolB, boolC);
+
+    ignoreUnused(charA, charB, charC);
+
+    ignoreUnused(numberA);
+    ignoreUnused(numberB);
+    ignoreUnused(numberC);
+
+    ignoreUnused(floatA);
+    ignoreUnused(floatB);
+    ignoreUnused(floatC);
+
+    ignoreUnused(doubleA);
+    ignoreUnused(doubleB);
+    ignoreUnused(doubleC);
```

Comment:
- [ ] pass all your variables to a single call to `ignoreUnused` just like you've done in each function below.